lib/virt_autotest/virtual_network_utils.pm

```diff
@@ -97,6 +97,7 @@ sub test_network_interface {
         }
```

Comment:
I did not noticed when this line" if (!(get_var('VIRT_AUTOTEST'))) {" was added for the block. Does it mean we will not restart network services any more? If so, "if (!get_var("SRIOV_NETWORK_CARD_PCI_PASSSHTROUGH")) {" is not required as well.

Reply:
Great, please help me make sure this as your mentioned.

Reply:
I need this line to set up the configuration file, and next line to bring the nic up, "ssh root\@$guest ifup $nic".

Reply:
@Julie-CAO thank you so much for your great help and reminder. 

updated code for QAM and SRIOV test here. 

Add the latest validation run:
gi-guest_developing-on-host_developing-kvm
10.67.131.5/tests/59

Reply:
LGTM